src/BuildingRegistry.Api.BackOffice.Abstractions/Validation/MoveBuildingUnit.cs

```diff
@@ -20,6 +20,14 @@ public static class BuildingInvalidStatus
 
                 public static TicketError ToTicketError() => new(Message, Code);
             }
+
+            public static class SourceAndDestinationBuildingAreTheSame
+            {
+                public const string Code = "BrongebouwIdHetzelfdeAlsDoelgebouwId";
+                public static string Message(string doelgebouwId) => $"Het brongebouwId is hetzelfde als het doelgebouwId: {doelgebouwId}.";
```

Comment:
In other messages we put the given value (doelgebouwId) between singular quotes, like so:
`Het gebouwId '{puri}' is niet gekend in het gebouwenregister.`
Shouldn't it be the same format for consistency sake? Like so:
`Het brongebouwId is hetzelfde als het doelgebouwId '{doelgebouwId}'.`
or
`Het brongebouwId '{doelgebouwId}' is hetzelfde als het doelgebouwId.`

Reply:
good catch, should change that!